db/schema.rb

```diff
@@ -10,21 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170525101357) do
+ActiveRecord::Schema.define(version: 20170525132019) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "dishes", force: :cascade do |t|
-    t.string   "name"
-    t.string   "description"
-    t.money    "price",       scale: 2
-    t.datetime "created_at",            null: false
-    t.datetime "updated_at",            null: false
-    t.integer  "menu_id"
-    t.index ["menu_id"], name: "index_dishes_on_menu_id", using: :btree
-  end
-
```

Comment:
@tochman before pushing the latest commit. I pulled the latest merge from develop. No migration file exist in the develop branch to update the schema with this line of code.

I believe the other team made some changes to the migration files Dishes as it's no longer available in the develop branch.